### PR TITLE
zh-translation: docs/reference/config/analysis/ist0109/index.md

### PR DESCRIPTION
**Pull request by orangegzx**
[ ] Configuration Infrastructure
[x] Docs
[ ] Installation
[ ] Networking
[ ] Performance and Scalability
[ ] Policies and Telemetry
[ ] Security
[ ] Test and Release
[ ] User Experience
[ ] Developer Infrastructure

### CODE REVIEW

**Comment by orangegzx** [comment]
@googlebot I signed it!

File: content/zh/docs/reference/config/analysis/ist0109/index.md [hunk 1]
```diff
@@ -57,8 +49,7 @@ spec:
 ---
 {{< /text >}}
 
-You can resolve this issue by setting the `exportTo` field to `.` so
-that each virtual service is scoped only to its own namespace:
+你可以通过设置 `exportTo` 字段为 `.` 来解决该问题，让每个虚拟服务都只限定在自己的命名空间：
```
Comment (kebe7jun) [inline content/zh/docs/reference/config/analysis/ist0109/index.md:52]:
```suggestion
您可以通过设置 `exportTo` 字段为 `.` 来解决该问题，让每个虚拟服务都只限定在自己的命名空间：
```

File: content/zh/docs/reference/config/analysis/ist0109/index.md [hunk 2]
```diff
@@ -1,32 +1,26 @@
...
+test: no
 ---
 
-This message occurs when Istio detects an overlap between
-[virtual service](/zh/docs/reference/config/networking/virtual-service)
-resources that conflict with one another. For example, multiple virtual
-services defined to use the same hostname and attached to a mesh gateway
-will generate an error message. Note that Istio supports merging of virtual
-services that are attached to the ingress gateways.
+当 Istio 检测到因[虚拟服务](/zh/docs/reference/config/networking/virtual-service)资源重复而导致冲突时，会出现该信息。比如，多个虚拟服务使用相同的主机名且连接网格网关，会出现错误信息。需要注意的是，Istio 支持虚拟服务合并来连接入口网关。
```
Comment (kebe7jun) [inline content/zh/docs/reference/config/analysis/ist0109/index.md:8]: 专有名词不翻译。
```suggestion
当 Istio 检测到因[Virtual Service](/zh/docs/reference/config/networking/virtual-service)资源重复而导致冲突时，会出现该信息。比如，多个 Virtual Service 使用相同的主机名且连接 Gateway，会出现错误信息。需要注意的是，Istio 支持 Virtual Service 合并来连接入口网关。
```

File: content/zh/docs/reference/config/analysis/ist0109/index.md [hunk 3]
```diff
@@ -1,32 +1,26 @@
...
-## Resolution
+## 解决方案{#resolution}
 
-To resolve this issue, you can take one of the following actions:
+解决该问题，有如下几个方法：
 
-* Merge the conflicting virtual services into a single resource.
-* Make the hostnames unique across virtual services attached to a mesh gateway.
-* Scope the resource to a specific namespace by setting the `exportTo` field.
+* 将冲突的虚拟服务合并为一个资源
```
Comment (kebe7jun) [inline content/zh/docs/reference/config/analysis/ist0109/index.md:14]:
```suggestion
* 将冲突的 Virtual Service 合并为一个
```

File: content/zh/docs/reference/config/analysis/ist0109/index.md [hunk 4]
```diff
@@ -1,32 +1,26 @@
...
+* 连接网格网关的虚拟服务使用唯一的主机名
```
Comment (kebe7jun) [inline content/zh/docs/reference/config/analysis/ist0109/index.md:15]:
```suggestion
* 连接 Gateway 的 Virtual Service 使用唯一的主机名
```

File: content/zh/docs/reference/config/analysis/ist0109/index.md [hunk 5]
```diff
@@ -1,32 +1,26 @@
...
+* 通过设置 `exportTo` 字段，将资源范围限定到指定的命名空间。
 
-## Examples
+## 示例{#examples}
 
-The `productpage` virtual service in namespace `team1` conflicts with the
-`custom` virtual service in `team2` namespace because both of the following
-are true:
+命名空间 `team1` 的虚拟服务 `productpage` 与命名空间 `team2` 的虚拟服务 `custom` 存在冲突的原因如下：
```
Comment (kebe7jun) [inline content/zh/docs/reference/config/analysis/ist0109/index.md:20]:
```suggestion
命名空间 `team1` 的虚拟服务 `productpage` 与命名空间 `team2` 的 Virtual Service `custom` 存在冲突的原因如下：
```

File: content/zh/docs/reference/config/analysis/ist0109/index.md [hunk 6]
```diff
@@ -1,32 +1,26 @@
...
 
-* They are attached to the default "mesh" gateway as no custom gateway is
-specified.
-* They both define the same host `productpage.default.svc.cluster.local`.
+* 因为没有指定自定义网关，它们被连接默认的“网格”网关。
```
Comment (kebe7jun) [inline content/zh/docs/reference/config/analysis/ist0109/index.md:22]:
```suggestion
* 因为没有指定自定义 Gateway，它们被连接默认的 Gateway。
```

File: content/zh/docs/reference/config/analysis/ist0109/index.md [hunk 7]
```diff
@@ -57,8 +51,7 @@ spec:
 ---
 {{< /text >}}
 
-You can resolve this issue by setting the `exportTo` field to `.` so
-that each virtual service is scoped only to its own namespace:
+您可以通过设置 `exportTo` 字段为 `.` 来解决该问题，让每个虚拟服务都只限定在自己的命名空间：
```
Comment (kebe7jun) [inline content/zh/docs/reference/config/analysis/ist0109/index.md:54]:
```suggestion
您可以通过设置 `exportTo` 字段为 `.` 来解决该问题，让每个 Virtual Service 都只限定在自己的命名空间：
```

**Review by kebe7jun — APPROVED** [review]
LGTM

**Pull request merged**